src/utils/helper.js

```diff
@@ -0,0 +1,59 @@
+
+export const FilterBysearch=(productList,{searchProduct})=>{
+    let updatedList=[...productList];
+    if(searchProduct){
+        return updatedList.filter((product)=>product.title.toLowerCase().includes(searchProduct.toLowerCase()))
+    }
+    return updatedList;
+}
+export const sortByPrice=(productList,{sortByPrice})=>{
+    let updatedList=[...productList];
+        if (sortByPrice === "HIGH_TO_LOW") {
+            return updatedList.sort((a, b) => b["price"] - a["price"]);
```

Comment:
you can directly use price for example  b.price - a.price . you don't need to use [""]